src/components/TodoItem.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import cn from 'classnames';
+import { TodosContext } from './TodosContext';
+import Todo from '../types/Todo';
+
+type Props = {
+  getTodo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ getTodo }) => {
+  const {
+    todos,
+    todoEditId,
+    todoEdit,
```

Comment:
do we need to save it in context if we use it only in this component, I don't think so